### PR TITLE
Fix #8495: Update pipeline sources for pipeline as edge lineage

### LINKED ISSUE #8495: Update pipeline sources for pipeline as edge lineage
To view pipeline as a edge metadata pipeline sources code to create lineage needs to updated.

By referring to work done for https://github.com/open-metadata/OpenMetadata/pull/5891 adapt the changes for below pipeline sources

- [ ] Airflow
- [ ] Airbyte
- [ ] Domo (Check if lineage exists for the source)
- [ ] Dagster (Check if lineage exists for the source)

### PR DESCRIPTION
**Pull request by ulixius9**
### Describe your changes :

Fix #8495: Update pipeline sources for pipeline as edge lineage

#
### Type of change :

- [x] Bug fix
- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

@open-metadata/ingestion

### CODE REVIEW

File: ingestion/src/metadata/utils/helpers.py [hunk 1]
```diff
@@ -325,3 +326,12 @@ def get_entity_tier_from_tags(tags: list[TagLabel]) -> Optional[str]:
         ),
         None,
     )
+
+
+def generate_http_basic_token(username, password):
```
Comment (pmbrull) [inline ingestion/src/metadata/utils/helpers.py:331]: should we move this to `credentials.py` or something?

File: openmetadata-spec/src/main/resources/json/schema/entity/services/connections/pipeline/airbyteConnection.json [hunk 2]
```diff
@@ -26,6 +26,17 @@
       "type": "string",
       "format": "uri"
     },
+    "username": {
```
Comment (pmbrull) [inline openmetadata-spec/src/main/resources/json/schema/entity/services/connections/pipeline/airbyteConnection.json:29]: let's add this to the 0.13 doc about the updates on the connections

**Review by pmbrull — APPROVED** [review]
Thanks @ulixius9 , leaving the approval. Added a couple of quick comments if you have the time. Let's fix the checkstyle and we can bring this in.
Thanks!

File: openmetadata-spec/src/main/resources/json/schema/entity/services/connections/pipeline/airbyteConnection.json (hunk 2 above)
Comment (ulixius9) [reply openmetadata-spec/src/main/resources/json/schema/entity/services/connections/pipeline/airbyteConnection.json:29]: sure will do that with #8502

**Review by pmbrull — APPROVED** [review]

**Pull request merged**